webapp/src/js/controllers/contacts.js

```diff
@@ -299,6 +299,12 @@ const PAGE_SIZE = 50;
         };
         setActionBarData();
         return ctrl.search();
+      })
+      .catch(err => {
+        $log.error('Error initializing contacts controller', err);
+        ctrl.error = true;
+        ctrl.loading = false;
+        ctrl.appending = false;
```

Comment:
Did you manage to reproduce this error somehow?

Reply:
Kind of... previous to this patch the "Contacts" tab was displaying the error message but if you tried to reload the all page you got the infinite spinner along with many parts of the UI not working well:

![Screenshot_20200708_123129](https://user-images.githubusercontent.com/1608415/86938782-2ce78380-c117-11ea-93a2-804eb2bd19e9.png)

Now with the patch at least an error is shown but still you see the lack of components that are not directly related with the patch:

![Screenshot_20200708_120950](https://user-images.githubusercontent.com/1608415/86936577-92864080-c114-11ea-9cc0-92ed3c82d545.png)